IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -75,41 +76,40 @@ + (MSIDMacTokenCache *)defaultCache
 
 - (nullable NSData *)serialize
 {
-    if (!self.cache)
+    if (!_cache)
     {
         return nil;
     }
+
+    __block NSData *result = nil;
     
-    __block NSDictionary *cacheCopy = nil;
     dispatch_sync(self.synchronizationQueue, ^{
-        cacheCopy = [self.cache mutableCopy];
+        NSDictionary *cacheCopy = [_cache mutableCopy];
```

Comment:
is this a shallow copy or deep copy?

Reply:
shallow

Reply:
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Collections/Articles/Copying.html